ogre2/src/Ogre2RenderEngine.cc

```diff
@@ -1343,7 +1343,7 @@ NativeWindowPtr Ogre2RenderEngine::CreateNativeWindow(
 #ifdef _WIN32
     params["externalGLContext"] =
       std::to_string((size_t)wglGetCurrentContext());
-#  elidef __APPLE__
+#elif defined(__APPLE__)
```

Comment:
this clearly fixed a typo, but it seems to have broken the macOS build. I'm not sure what it should be

Reply:
Then it seems nothing should be done on Apple? My reasoning:

1. If macOS understood elidef, it would have failed anyways.
2. If macOS did not understand elidef, then it did not execute the branch after elidef (because it is "protected" by `ifdef _WIN32`).

Reply:
Resp. it should most probably fall to the `else` branch...

Reply:
The homebrew CI runs ogre2 + metal on apple so does not execute this code path. I think ogre2 + opengl is rarely used now and may not be fully working..

Reply:
Oh, @mjcarroll was faster :) https://github.com/gazebosim/gz-rendering/pull/911 . Thanks!